python/variable.cpp

```diff
@@ -335,4 +336,33 @@ Mostly equivalent to Variable, see there for details.)");
                    "values, False otherwise.")
           .rtype("bool")
           .c_str());
+
+  m.def(
+      "is_sorted",
+      [](const VariableConstView &x, const Dim dim, const std::string &order) {
+        if (order == "ascending")
+          return is_sorted(x, dim, variable::SortOrder::Ascending);
+        else if (order == "descending")
```

Comment:
See comment alongside `acccumlate_in_place` implementation: This is what I expected on the c++ side also.

Reply:
On the C++ side we have an enum, so having wrong values actually requires changing the enum. On the Python side we are using a string, so handling the very possible case of bad inputs is imho much more important?